scripts/test.sh

```diff
@@ -410,17 +410,8 @@ function unconvert_pass {
   run_for_modules generic_checker run_go_tool "github.com/mdempsky/unconvert" unconvert -v
 }
 
-function ineffassign_per_package {
-  # bash 3.x compatible replacement of: mapfile -t gofiles < <(go_srcs_in_module)
-  local gofiles=()
-  while IFS= read -r line; do gofiles+=("$line"); done < <(go_srcs_in_module)
-
-  # TODO: ineffassign should work with package instead of files
-  run_go_tool github.com/gordonklaus/ineffassign "${gofiles[@]}"
-}
-
 function ineffassign_pass {
-  run_for_modules generic_checker ineffassign_per_package
+  run_for_modules generic_checker run_go_tool "github.com/gordonklaus/ineffassign"
```

Comment:
What's your plan for `ineffassign_pass` if this will be covered by `make verify-lint`? Should we maybe remove it?

Reply:
```shell
# TODO: https://github.com/etcd-io/etcd/issues/16610
#
# The golangci-lint doesn't verify sub modules. Before #16610 fixed, verify-lint
# still depends on legacy {ineffassign,nakedret,unparam,...}_pass. These X_pass
# will be removed when the golangci-lint covers all the sub modules.
.PHONY: verify-lint
verify-lint: verify-ineffassign
	golangci-lint run --config tools/.golangci.yaml
```

I add new recipe `verify-ineffassign` and `verify-lint` depends on it. So, it can cover the new change.
when all the lint issues are covered by `_pass`, we can switch to golangci-lint. After that, all the legacy lint_pass can be removed. Does it make senses?

Reply:
Make sense.

Reply:
> when all the lint issues are covered by _pass, we can switch to golangci-lint. After that, all the legacy lint_pass can be removed. 

I assume there is a ticket to track this.

Reply:
Yes. https://github.com/etcd-io/etcd/issues/16610 is tracking this

Reply:
thx